vas2nets/tasks.py

```diff
@@ -0,0 +1,29 @@
+import csv
+import requests
+
+from django.conf import settings
+from celery.task import Task
+
+from .models import VoiceCall
+
+
+class FetchVoiceData(Task):
+
+    def get_data(self, date):
+        url = "%s?report_date=%s" % (settings.V2N_VOICE_URL, date)
+
+        content = requests.get(url).content.decode('utf-8')
+
+        return list(csv.reader(content.splitlines(), delimiter=','))[1:]
```

Comment:
Maybe we can use a DictReader here instead? That way if they reorder/add columns, this will still work fine.